scrub_hub_project/authenticate/views.py

```diff
@@ -19,7 +20,7 @@ def login_view(request):
         data = json.loads(request.body)
```

Comment:
Generally functions should not return multiple types, because this is really bad practice. API calls and web pages are two different things, and thus should be separate views for better clarity :)